build/CopyDependencies/CopyDependencies.targets

```diff
@@ -0,0 +1,29 @@
+﻿<Project>
+  <Import Project="..\..\src\EmbeddedSonarAnalyzer.props" />
+
+  <Target Name="Build">
+    <CallTarget Targets="ClearDirs" />
+    <CallTarget Targets="CopyJars" />
+  </Target>
+
+  <PropertyGroup>
+    <JarDownloadDir>$(LOCALAPPDATA)\SLVS_Build_DownloadedJars</JarDownloadDir>
+    <EmbeddedJarDir>..\..\src\Integration.Vsix\DownloadedJars</EmbeddedJarDir>
+  </PropertyGroup>
+  
+  
+  <Target Name="ClearDirs">
+    <RemoveDir Directories="$(EmbeddedJarDir)" />
+  </Target>
+
+  <Target Name="CopyJars">
+    <ItemGroup>
+      <SourceJars Include="$(JarDownloadDir)\sonar-cfamily-plugin-$(EmbeddedSonarCFamilyAnalyzerVersion).jar" />
```

Comment:
Will this fail if it can't find any of those files?

Reply:
Yes, and I believe it's desired behaviour.